decidim-core/app/presenters/decidim/home_stats_presenter.rb

```diff
@@ -49,15 +49,15 @@ def not_highlighted
     private
 
     def global_stats(conditions)
-      Decidim.stats.except([:users_count, :processes_count])
+      Decidim.stats.except([:users_count, :proposals_count])
              .filter(conditions)
              .with_context(published_features)
              .map { |name, data| [name, data] }
     end
 
     def feature_stats(conditions)
-      Decidim.feature_manifests.map do |feature|
-        feature.stats.filter(conditions).with_context(published_features).map { |name, data| [name, data] }.flatten
+      Decidim.feature_manifests.flat_map do |feature|
```

Comment:
I'm not sure `flat_map` is equivalent to the previous `map{ ... }.flatten(1)` we used. Can you check it, please?